src/test/suite/fixtures/recorded/languages/csharp/changeCondition2.yml

```diff
@@ -0,0 +1,23 @@
+languageId: csharp
```

Comment:
would be good to also have a for-loop test where selection is in body of loop

Reply:
I've added two.